network-wrapper

```diff
@@ -292,5 +292,7 @@ if [ $DEBUG -eq 1 ] ; then
 fi
 # Remove NSPID to avoid `ip netns` catch it.
 [ -f /var/run/netns/$NSPID ] && rm -f /var/run/netns/$NSPID
+# Added a debug printout to let know that the script ran to completion successfully.
+echo network-wrapper configuration successful: $GUEST_LABEL_NAME
```

Comment:
The output should be json compatible, current output is json compatible, but your change is breaking that.

Reply:
The output is json compatible.  Have added a check based on the network-wrapper script output.